blog.html

```diff
@@ -0,0 +1,242 @@
+         </div>
+ 
+         <div class="item_movie">
+             <a target="_blank" href="https://www.youtube.com/watch?v=qZuQc-GvRlk"> <img src="./img/movie_blog/black panterr.jpg" alt="Black panterr:Wakanda forever" class="blog_img" title="Black panterr:Wakanda forever comming soon" ></a>
+         </div>
+ 
+         <div class="item_movie">
+             <a target="_blank" href="https://www.youtube.com/watch?v=BbXJ3_AQE_o">  <img src="./img/movie_blog/Spider man.jpg" alt="Spider man" class="blog_img" title="Spider man comming soon" ></a>
+         </div>
+ 
+         <div class="item_movie">
+             <a target="_blank" href="https://www.youtube.com/watch?v=KHPrDP0F5_o">  <img src="./img/movie_blog/Toor.jpg" alt="thor 2"class="blog_img" title=" THOR comming soon"></a>
+         </div>
+      </div>
+        </div>
+      
+    </div>
+    <h1 class="blog_title">THE MOST DISCUSSED</h1>
+    <div class="discussed">
+        <div class="discussed_movies">
+           <a href="https://www.youtube.com/watch?v=-zrO28TkhPY" target="_blank">  <img src="./img/movie_blog/2+1.jpg" alt="2+1" class="discussed_movies_item" ></a>
+
+        </div>
+        
+        <div>
+            
+            <p style="margin-top: 10%;">
+                Black handsome Samuel leads a fun life as a yacht captain, arranging fun swims for money. <br>
+                True, the yacht belongs to Samantha, but Samuel's charm allows the hero to break the rules and<br>
+                 often throw parties. One day, a girl with a child appears on the pier, and after a short<br>
+                  conversation it turns out that Gloria is Samuel's three-month-old daughter. Kristin deceives <br>
+                   the guy and, leaving Gloria, flies to London. Not wanting to become a single father, and indeed<br>
+                    the father of a child, Samuel goes to London, wanting to find Christina and return the girl.<br>
+                     However, it turns out that the information about Christina's work is false. Upon learning that <br>
+                      her captain left the country without permission, Samantha dismisses Samuel and he remains in <br>
+                       London without money and with a child in his arms. The hero is unexpectedly lucky when he sees how<br>
+                        Samuel arranges a performance at the subway station so as not to lose Gloria, producer Bernie <br>
+                         offers the former captain a job as a stuntman. The contract turns out to be so profitable that in<br>
+                          the following years Gloria's life turns into a solid holiday of childhood. However, ahead of Gloria<br>
+                           and Samuel are waiting for life's trials and the unexpected appearance of the missing Christina.<br>
+                            Whether the heroes will be able to overcome the trials, see the drama film "2 + 1".
```

Comment:
If you wrote your text with `<br/>`s to just move the text to the next line, you can just remove them, because `<p>` will automatically fit the text inside the container. If you need to specify the length of the lines, you can change the sizes of the paragraph itself. I've made a demo for this: https://codesandbox.io/s/festive-tree-crgto?file=/index.html